ApprovalTests.Xunit2/CompatibilityWithXunit1AsyncFixtureTest.cs

```diff
@@ -0,0 +1,52 @@
+using ApprovalTests.Reporters;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+using static ApprovalTests.Xunit2.CompatibilityWithXunit1AsyncFixutureTest;
+
+namespace ApprovalTests.Xunit2
+{
+    public class CompatibilityWithXunit1AsyncFixutureTest : IClassFixture<DummyTestStateClass>
+    {
+        public class DummyTestStateClass : IDisposable
+        {
+            public string ConnectionString { get; }
+
+            public DummyTestStateClass()
+            {
+                ConnectionString = "foo";
+            }
+
+            public void Dispose() { }
+        }
+
+        private string _connectionString;
+
+        public CompatibilityWithXunit1AsyncFixutureTest(DummyTestStateClass state)
+        {
+            _connectionString = state.ConnectionString;
+        }
+
+        [Fact]
+        [UseReporter(typeof(DiffReporter))]
+        public async void Xunit2ShouldWorkWithAsyncFixtureTest()
```

Comment:
shouldnt this return a Task?

Reply:
I honestly don't know, I created that PR in February and I can't remember. Let me know if you need me to look at it if you have not fixed it already.

Reply:
just took a look, and yes, I can't see a reason why it should return async void. All I could remember was testing some weird edge cases, and thought perhaps there was some strange quirk that this was needed. Looking at the code it looks like a simple mistake. The important part of the test is ensuring (testing) that we succesfully  call `Approvals.Verify` from within an async block. The reason I wrote these in the first place (phew... feels like a lifetime ago) was because I  discovered I couldnt use approvals at a banking client for our async tests. It was the only way I could  report the approvals problem. I don't even have the sln open, I'm on another machine. If this test is failing the build, is it possible the problem still exists? i.e. it's not failing because it returns void (false negative), but failing because it's a positive negative. I say this because I saw that the PR request is failing a build and I have not even opened the project to look. So, yes, it should return `Task`.

Reply:
yes it should.